### PR TITLE
Added dropdown menu in home page

### PR DESCRIPTION
**Pull request by dsnehasish74**
Issue:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,274 +1,240 @@
...
-    />
-    <link href="assets/css/main.css" rel="stylesheet" />
-    <link href="assets/css/header.css" rel="stylesheet" />
-    <link href="assets/css/footer.css" rel="stylesheet" />
-    <link href="assets/css/header.css" rel="stylesheet" />
-    <link href="assets/css/variables.css" rel="stylesheet" />
-    <link href="dist/sbuttons.css" rel="stylesheet" />
-    <link href="assets/css/index.css" rel="stylesheet" />
-    <link href="assets/css/documentation.css" rel="stylesheet" />
-    <link href="assets/css/carbon.css" rel="stylesheet" />
```
Comment (shahednasser) [inline index.html:3]: Remove this additional line

**Review by shahednasser — CHANGES_REQUESTED** [review]
Great work @dsnehasish74 I just need you to make the following changes:

1. can you move the arrow next to the logo and add "Menu" as a title to it as well? That way it wouldn't be confusing.
2. Change the color of the dropdown divider to #42668a
3. When the dropdown is opened on a small device and the screen is enlarged, the dropdown is still visible. it should close.

**Comment by dsnehasish74** [comment]
Ok I am doing it

File: assets/js/index.js [hunk 2]
```diff
@@ -348,6 +348,11 @@ $(document).ready(function () {
   $(".dropdown-toggle").click(function (event) {
     $(".dropdown-menu").toggle();
   });
-
   //For dropdown menu
 });
+
+$(window).resize(function () {
+  if ($(this).width() < 767) {
```
Comment (shahednasser) [inline assets/js/index.js:355]: This should be greater than 768

File: assets/css/header.css [hunk 3]
```diff
@@ -300,15 +300,23 @@ a:hover {
...
   transition: all 0.5s;
 }
 .dropdown-item:hover {
   background-color: #324f6b;
 }
+.dropdown-item {
+  color: rgba(235, 235, 235, 0.815);
+}
+.dropdown-divider {
+  background-color: #42668a;
```
Comment (shahednasser) [inline assets/css/header.css:314]: this should be the color of the border not background

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: assets/js/index.js [hunk 4]
```diff
@@ -352,7 +352,7 @@ $(document).ready(function () {
 });
 
 $(window).resize(function () {
-  if ($(this).width() < 767) {
+  if ($(this).width() < 768) {
```
Comment (shahednasser) [inline assets/js/index.js:355]: Should be greater not less

**Comment by shahednasser** [comment]
Great thank you!

**Pull request merged**